### DISCUSSION TITLE
GroupChat Messages not working?

### DISCUSSION THREAD

**Discussion by WebsheetPlugin** [microsoft/autogen#2518] [Q&A]
0.2.26
messages = [{'content': 'what can you do?', 'role': 'user', 'name': 'USER'}, {'content': 'I can do the following tasks...', 'role': 'assistant', 'name': 'USER'}, {'role': 'user', 'content': 'what does 3?', 'name': 'USER'}]

groupchat = autogen.GroupChat(
  agents=[agent1, agent2],
  messages=messages,
  max_round=12,
  speaker_selection_method=self.custom_speaker_selection_func,
)
manager = autogen.GroupChatManager(groupchat=groupchat)

I tried:
manager.initiate_chat(user_proxy, clear_history=False)
and
user_proxy.initiate_chat(manager, clear_history=False)

I have tried many ways but I can not make messages work inside groupchat. Only way for me currently is to use transform messages. 

Also manager.run_chat() is an mystery to me. As it is also not working and has no working example onlinne.

1. When I set messages trough groupchat should they be added to the agents during their conversations? that's the idea, right?
2. Do messages require "name" ?
3. Does anyone has this really working?
4. Is there anywhere an working example?

**Comment by ekzhu** [comment]
```python
user_proxy.initiate_chat(manager, clear_history=False)
```

This is how all examples use group chat. 

Can you describe what is the issue you are facing? What is not working?

**Comment by WebsheetPlugin** [comment]
I refer to "messages" parameter. I am not able to make it work in any way. 

There is no example of using the "messages" parameter. I don't think this works. Or I don't know how it's supposed to work..

**Reply by ekzhu** [reply]
Got it. We are going to address this in #2359. Could you comment on this issue, and we can add you as a reviewer?
